backend/index.js

```diff
@@ -10,8 +13,34 @@ const wordRoutes = require("./routes/words");
 const app = express();
 const PORT = process.env.PORT || 8080;
 
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    cb(null, "./images");
+  },
+  filename: (req, file, cb) => {
+    const uniqueSuffix = Date.now() + "-" + Math.round(Math.random() * 1e9);
+    cb(null, `${file.fieldname} - ${uniqueSuffix}`);
+  },
+});
+
+const fileFilter = (req, file, cb) => {
+  if (
+    file.mimetype == "image/png" ||
+    file.mimetype == "image/jpg" ||
+    file.mimetype == "image/jpeg"
+  ) {
+    cb(null, true);
+  } else {
+    cb(null, false);
+    return cb(new Error("Only .png, .jpg and .jpeg format allowed!"));
+  }
+};
```

Comment:
- move these two functions to separate file under `hlepers` directory. Something like `helpers\fileHelper.js`